.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16'
+          node-version: '16.20.2'
```

Comment:
What benefit does setting a specific version here give?

Reply:
no specific version are necessary, i was just testing with the latest version of the node 16; changing to major version 16 for aa clean state.